src/components/review/AssignTechnicalReview.tsx

```diff
@@ -0,0 +1,117 @@
+import { Button, makeStyles, TextField } from '@material-ui/core';
+import { Autocomplete } from '@material-ui/lab';
+import React, { useState } from 'react';
+
+import { BasicUserDetails, Proposal, UserRole } from 'generated/sdk';
+import { useUsersData } from 'hooks/user/useUsersData';
+import useDataApiWithFeedback from 'utils/useDataApiWithFeedback';
+import withConfirm, { WithConfirmType } from 'utils/withConfirm';
+
+type AssignTechnicalReviewProps = {
+  proposal: Proposal;
+  onProposalUpdated: (proposal: Proposal) => void;
+  confirm: WithConfirmType;
+};
+
+const getName = (user: BasicUserDetails | undefined) =>
+  `${user?.firstname} ${user?.lastname}`;
+
+const useStyles = makeStyles((theme) => ({
+  userList: {
+    width: '300px',
+    marginTop: theme.spacing(3),
+    display: 'inline-block',
+  },
+  submitButton: {
+    marginLeft: theme.spacing(2),
+  },
+}));
+function AssignTechnicalReview({
+  proposal,
+  onProposalUpdated,
+  confirm,
+}: AssignTechnicalReviewProps) {
+  const classes = useStyles();
+  const { api } = useDataApiWithFeedback();
+
+  const [selectedUser, setSelectedUser] = useState(
+    proposal.technicalReviewAssignee
+  );
+  const { usersData } = useUsersData({
+    userRole: UserRole.INSTRUMENT_SCIENTIST,
+  });
+
+  if (!usersData) {
+    return null;
```

Comment:
Isn't it better to return <UOLoader> instead of null or maybe not? I am not completley sure about the use of the component it is just like a question but maybe it already has loader in this screen.

Reply:
If it's just `return` then it is equal to `return undefined`. React to rener needs either `React.element` or `null`